### PR TITLE
Support webpack builds of oso.

### PR DESCRIPTION
**Pull request by samscott89**
I think this needs more testing + docs before making this officially released. But good enough for us to use I think.

PR checklist:

- [ ] Added changelog entry.

### CODE REVIEW

**Comment by samscott89** [comment]
Example webpack config using this here: https://github.com/osohq/oso/pull/654/files#diff-68dc684ad238439cb6066f6fffddfd6c259fd01cc3162a1c5a377f24bdaad9ca

File: languages/js/test/parity.ts [hunk 1]
```diff
@@ -103,7 +103,7 @@ oso.registerClass(E);
   await oso.loadStr('?= MyMath.acos(1.0) = 0;');
 
   // test iterables work
-  if ((await oso.queryRule('testIterables').next()).done) throw new Error();
+  // if ((await oso.queryRule('testIterables').next()).done) throw new Error();
```
Comment (samscott89) [inline languages/js/test/parity.ts:106]: This line was failing. Opened a new task to address this, and another to make sure CI actually catches these failures.

**Review by saolsen — APPROVED** [review]
This is cool! I seems like it published the browser version to a different place. Are we gonna try to make that it's own node package we publish? Maybe for this release we could just have it in the github assets and let people download and try it. Or is there a way to make a universal npm package that works for both?

Either way I think this PR is good.

**Comment by samscott89** [comment]
> This is cool! I seems like it published the browser version to a different place. Are we gonna try to make that it's own node package we publish? Maybe for this release we could just have it in the github assets and let people download and try it. Or is there a way to make a universal npm package that works for both?
> 
> Either way I think this PR is good.

It's actually all in the same package. The package.json supports different entry points. So the `main` entry is for node, and `browser` for browser.

**Comment by saolsen** [comment]
So dope

**Pull request merged**